src/lib/datepicker/datepicker.md

```diff
@@ -148,6 +155,12 @@ export class MyComponent {
 }
 ```
 
+*Please note: `MdNativeDateModule` is based off of the functionality available in JavaScript's
+native Date object, and is thus not suitable for many locales. One of the biggest shortcomings of
+the native Date object is the inability to set the parse format. We highly recommend writing a
+custom `DateAdapter` that works with the formatting/parsing library of your choice, or waiting for
+the official Moment.js module.*
```

Comment:
On moment: I would rephrase this to not convey that Moment is the "official" way of doing i18n, but just that we plan on making a moment adapter available as a convenience